style/ruby/README.md

```diff
@@ -0,0 +1,14 @@
+# Ruby Style Enforcement
+
+It is recommended that you use a ruby style enforcement tool such as [RuboCop](/bbatsov/rubocop). This can be integrated in your CI build and optionally use a service like HoundCI to comment on Pull Requests indicating violations.
```

Comment:
@ndushay there is a link to the RuboCop GitHub page.  I think all the information one would need is all there.